cardano-api/src/Cardano/Client/API.hs

```diff
@@ -0,0 +1,318 @@
+-- consensus uses byron types
+-- type-instantiations for codecs
+
+module Cardano.Client.API
+(
+-- Ouroboros.Network.Point
```

Comment:
You can turn those comment in to haddock section headers
```
-- * Ouroboros.Network.Point re-exports
```

It's also good to align them vertically with the rest of the list